src/components/button-toggle/button-toggle.stories.tsx

```diff
@@ -75,6 +157,24 @@ export const small: ComponentStory<typeof ButtonToggle> = () => (
   </ButtonToggleGroup>
 );
 
+export const smallMinor: ComponentStory<typeof ButtonToggle> = () => (
```

Comment:
**issue**: Doesn't look like this story has been included into `button-toggle.stories.mdx`

Reply:
**suggestion**: include story like so `story={stories.smallMinor}`(even though i'm against this approach)

Reply:
We should follow the naming convention so it should be `SmallMinor`